### PR TITLE
Obtain examples from bio-formats-examples and add bioformats 5.8.1 changes

### PR DESCRIPTION
**Pull request by rleigh-codelibre**
Check east-ci build is green and the docs artefact contains the examples.

Note: expected to fail travis/appveyor builds because there are no bio-formats-examples artefacts in artifactory.

### CODE REVIEW

**Comment by rleigh-codelibre** [comment]
Green in https://web-proxy.openmicroscopy.org/east-ci/job/BIOFORMATS-merge/54/

**Comment by rleigh-codelibre** [comment]
Example docs: https://web-proxy.openmicroscopy.org/east-ci/job/BIOFORMATS-merge/jdk=JDK9,label=testintegration/ws/bio-formats-build/bio-formats-documentation/target/bio-formats-documentation-5.8.1-SNAPSHOT-documentation.tar.gz

**Comment by dgault** [comment]
Build is green and the changes look OK however I do not have access to east-ci so @joshmoore may be best placed to check to built docs

**Comment by joshmoore** [comment]
Discussion today (updated in the `Bio-Formats decoupling and building` notes) suggests that we'll try to get this green by releasing a SNAPSHOT of examples to artifactory.

**Comment by rleigh-codelibre** [comment]
Updated to include the changes from https://github.com/ome/bio-formats-documentation/pull/11 and to use the 5.8.1 examples source release jar.

**Comment by rleigh-codelibre** [comment]
Build on east-ci; see https://web-proxy.openmicroscopy.org/east-ci/job/BIOFORMATS-merge/jdk=JDK9,label=testintegration/ws/bio-formats-build/bio-formats-documentation/target/ for docs zip

**Comment by rleigh-codelibre** [comment]
@hflynn If you wanted to have a look over this, this PR should make the decoupled docs the same as the official 5.8.1 bioformats release docs, and get the examples from the decoupled examples repository.

**Comment by hflynn** [comment]
I've just downloaded the zip to look at it @rleigh-codelibre

**Comment by hflynn** [comment]
I can't see any issues with this - not spotted any differences between the zip docs and https://docs.openmicroscopy.org/bio-formats/5.8.1/ and the included example code seems to be copied over fine e.g. on https://docs.openmicroscopy.org/bio-formats/5.8.1/developers/matlab-dev.html, https://docs.openmicroscopy.org/bio-formats/5.8.1/developers/tiling.html and https://docs.openmicroscopy.org/bio-formats/5.8.1/developers/in-memory.html

**Comment by joshmoore** [comment]
Great, thanks, @hflynn. Merging & tagging this as 5.8.1 then. We'll have one more chance during the 5.8.2 lifecycle to discover any changes that have gone missing.

File: pom.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
   <groupId>ome</groupId>
   <artifactId>bio-formats-documentation</artifactId>
-  <version>5.8.1-SNAPSHOT</version>
+  <version>5.8.1</version>
```
Comment (sbesson) [inline pom.xml]: Should this `version` be bumped to `5.8.2-SNAPSHOT` on master?

Comment (rleigh-codelibre) [reply pom.xml]: Yes, definitely now that 5.8.1 has been tagged.

**Pull request merged**